resources/public/api/conf/1.0/errors.raml

```diff
@@ -11,7 +11,7 @@ traits:
           application/json:
             type: types.errorResponse
             example:
-              description: The matching resource is not found.
+              description: The supplied income source could not be found.
```

Comment:
hmm this one is shared and applied to all endpoints. But the above is a change to a single endpoint for OAS
I think this change is for delete only? maybe we should just make the change only for that specific endpoint?